src/styles/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 32px 16px 16px;
+  margin-right: 48px;
+
+  &__image-block {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+    display: block;
+  }
+
+  &__review {
+    @extend %display-center;
+    margin-bottom: 24px;
+  }
+
+  &__value {
+    font-size: 10px;
+    color: $black;
+  }
+
+  &__description-title {
+    font-weight: 500;
+    font-size: $font-size-medium;
+    line-height: 18px;
+    margin: 40px 0 4px;
+  }
+
+  &__description-code {
+    font-size: 10px;
+    color: $light-gray;
+    margin-bottom: 16px;
+  }
+
+  &__price-name {
+    font-size: $font-size-medium;
+    color: $light-gray;
+  }
+
+  &__price {
+    @extend %display-center;
+    margin-bottom: 16px;
+  }
+
+  &__price-value {
+    font-weight: 700;
+    font-size: 16px;
+    color: $black;
+  }
+
+  &__button-link {
+    width: 100%;
+    text-transform: uppercase;
+    font-weight: 700;
+    font-size: 14px;
+    color: white;
+    transition: 0.3s;
```

Comment:
Better to define which exact properties will be animated, because if you not define it, it will be applied to every property that can be animated